README.md

```diff
@@ -115,9 +115,9 @@ Using jQuery, add an event listener for clicks on the button with the id
 body, that says "hello".
 
 Your Answer:
-```js
-// your code here
-```
+$("#greeting").click(function(){
+    $(body).append("<p>hello</p>");
```

Comment:
Missing just a little punctuation here. This would give you an error about body not being defined